tests/libc/host/host.cpp

```diff
@@ -47,13 +65,29 @@ int main(int argc, const char* argv[])
 
     printf("=== %s: %s\n", argv[0], argv[1]);
 
+#if defined(_WIN32)
+    printf("Windows Host - Initial value of MXCSR is %0x\n", my_getmxcsr());
+#endif
+
     // Create the enclave:
     if ((result = oe_create_libc_enclave(
              argv[1], OE_ENCLAVE_TYPE_SGX, flags, NULL, 0, &enclave)) != OE_OK)
         oe_put_err("oe_create_libc_enclave(): result=%u", result);
 
+#if defined(_WIN32)
+    printf(
+        "Windows Host - Value of MXCSR after create_libc_enclave is %0x\n",
+        my_getmxcsr());
```

Comment:
It would be good to OE_TEST the expected values.

Reply:
I was meaning to delete these. But like your idea.

Reply:
Done